patches/chrome-android-java-src-org-chromium-chrome-browser-externalnav-ExternalNavigationHandler.java.patch

```diff
@@ -0,0 +1,77 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/externalnav/ExternalNavigationHandler.java b/chrome/android/java/src/org/chromium/chrome/browser/externalnav/ExternalNavigationHandler.java
+index bf59138a49f47503d04ab1254fb68998655f7e41..180828114a771c581f5d19e60b3f97c7e41961ff 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/externalnav/ExternalNavigationHandler.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/externalnav/ExternalNavigationHandler.java
+@@ -29,6 +29,7 @@ import org.chromium.chrome.browser.ChromeSwitches;
+ import org.chromium.chrome.browser.IntentHandler;
+ import org.chromium.chrome.browser.customtabs.CustomTabIntentDataProvider.LaunchSourceType;
+ import org.chromium.chrome.browser.instantapps.InstantAppsHandler;
++import org.chromium.chrome.browser.preferences.website.PlayYTVideoInBrowserPreferences;
```

Comment:
Trying to avoid patching.

Reply:
I don't see a goo patch here as well, but perhaps at least we can move all functions like `playYTVideoInBrowserEnabled` `ContainsYT` and the code that checks inside a different java file Brave owned and just one string patch call them. @bridiver do you have a better idea?

Reply:
My idea is adding our own NavigationThrottle. will try this today.